### PR TITLE
updated analysis annotations

### PR DESCRIPTION
**Pull request by blogsdon**
Completely updated analysis annotations with descriptions and references filled in.

### CODE REVIEW

**Comment by ychae** [comment]
Can we add PR #150 (transcriptNormalizationMethod: fpkm and TMM) to this PR and close 150?

File: synapseAnnotations/data/analysis.json [hunk 1]
```diff
@@ -1,388 +1,340 @@
...
+    "name": "assemblyMethod",
+    "description": "Algorithm used to build a reference genome from short read sequencing data.",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "velvet", 
-        "description": "", 
-        "source": ""
+        "value": "velvet",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Velvet is capitalized as if it's a proper noun in the source. Though maybe we just don't care.

File: synapseAnnotations/data/analysis.json [hunk 2]
```diff
@@ -1,388 +1,340 @@
...
+    "description": "Methods for identifying or correcting potential confounding variables in regression analyses.",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "COMBAT", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "COMBAT",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: I can't find "COMBAT" in the source. Can someone else confirm the source is correct?

Comment (teslajoy) [reply synapseAnnotations/data/analysis.json]: This is what I went by in grad-school 
https://www.bu.edu/jlab/wp-assets/ComBat/Abstract.html

File: synapseAnnotations/data/analysis.json [hunk 3]
```diff
@@ -1,388 +1,340 @@
...
-      }, 
+        "value": "surrogate variable analysis",
+        "description": "Surrogate Variable Analysis: identification and adjustment for hidden confounds that are independent of a known outcome",
+        "source": "http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030161"
+      },
       {
-        "value": "PCA", 
-        "description": "", 
-        "source": ""
+        "value": "principal component analysis",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: "Preferred Name" is "Principal Component Analysis"

File: synapseAnnotations/data/analysis.json [hunk 4]
```diff
@@ -1,388 +1,340 @@
...
+        "value": "k-means clustering",
+        "description": "A k-means clustering is a data transformation which achieves a class discovery or partitioning objective, which takes as input a collection of objects (represented as points in multidimensional space) and which partitions them into a specified number k of clusters. The algorithm attempts to find the centers of natural clusters in the data. The most common form of the algorithm starts by partitioning the input points into k initial sets, either at random or using some heuristic data. It then calculates the mean point, or centroid, of each set. It constructs a new partition by associating each point with the closest centroid. Then the centroids are recalculated for the new clusters, and the algorithm repeated by alternate applications of these two steps until convergence, which is obtained when the points no longer switch clusters (or alternatively centroids are no longer changed).",
+        "source": "http://purl.obolibrary.org/obo/OBI_0200041"
+      },
       {
-        "value": "WGCNA", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "WGCNA",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Is a synonym of "Preferred Name": "Weighted correlation network analysis"

File: synapseAnnotations/data/analysis.json [hunk 5]
```diff
@@ -1,532 +1,479 @@
...
+        "description": "A data mining method typically used for studying biological networks based on pairwise correlations between variables.",
+        "source": "http://edamontology.org/operation_3766"
+      },
       {
-        "value": "principalComponentAnalysis", 
-        "description": "", 
-        "source": ""
+        "value": "principal component analysis",
```
[the remark this answers is not in the document]
Comment (philerooski) [reply synapseAnnotations/data/analysis.json]: "Preferred Name" is "Principal Component Analysis"

File: synapseAnnotations/data/analysis.json [hunk 6]
```diff
@@ -1,388 +1,340 @@
...
-        "value": "varDict", 
-        "description": "", 
-        "source": ""
-      }, 
-      {
-        "value": "varDict2", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "varDict",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Should be "VarDict"

File: synapseAnnotations/data/analysis.json [hunk 7]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "A DNA and RNA seq variant caller in cancer research.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/27060149"
+      },
       {
-        "value": "mutect", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "mutect",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Should be "MuTect"

File: synapseAnnotations/data/analysis.json [hunk 8]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "A method to detect DNA point mutations in heterogeneous and impure cancer samples.",
+        "source": "https://www.nature.com/articles/nbt.2514"
+      },
       {
-        "value": "varScan", 
-        "description": "", 
-        "source": ""
+        "value": "varScan",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Should be "VarScan"

File: synapseAnnotations/data/analysis.json [hunk 9]
```diff
@@ -1,388 +1,340 @@
...
+    "description": "",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "WGCNA", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "WGCNA",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Preferred Name: Weighted correlation network analysis

File: synapseAnnotations/data/analysis.json [hunk 10]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "A data mining method typically used for studying biological networks based on pairwise correlations between variables.",
+        "source": "http://edamontology.org/operation_3766"
+      },
       {
-        "value": "sparrow", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "sparrow",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: All capitalized. "SPARROW".

File: synapseAnnotations/data/analysis.json [hunk 11]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Sparse conditional independence graph construction from gene expression data using a variational Bayes spike regression neighorhood selection method.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/?term=25583238"
+      },
       {
-        "value": "lasso", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "lasso",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Lasso

File: synapseAnnotations/data/analysis.json [hunk 12]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Sparse conditional independence graph construction from gene expression data using an L1 penalized regression neighborhood selection method.",
+        "source": "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-384"
+      },
       {
-        "value": "ridge", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "ridge",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: The source has both "Ridge Regression", "ridge regression"

File: synapseAnnotations/data/analysis.json [hunk 13]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Conditional independence graph construction from gene expression using an L2^2 penalized regression neighborhood selection method.",
+        "source": "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-384"
+      },
       {
-        "value": "genie3", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "genie3",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: GENIE3

File: synapseAnnotations/data/analysis.json [hunk 14]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Gene expression network reconstruction using random forests.",
+        "source": "http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0012776"
+      },
       {
-        "value": "tigress", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "tigress",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: TIGRESS

File: synapseAnnotations/data/analysis.json [hunk 15]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Gene expression network reconstruction using stability selection.",
+        "source": "https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-6-145"
+      },
       {
-        "value": "aracne", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "aracne",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: ARACNE

File: synapseAnnotations/data/analysis.json [hunk 16]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Gene expression network reconstruction using mutual information.",
+        "source": "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-S1-S7"
+      },
       {
-        "value": "mrnet", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "mrnet",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: MRNET

File: synapseAnnotations/data/analysis.json [hunk 17]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "An information theoretic method to infer gene expression networks.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/18354736"
+      },
       {
-        "value": "c3net", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "c3net",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: C3NET

File: synapseAnnotations/data/analysis.json [hunk 18]
```diff
@@ -1,388 +1,340 @@
...
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "limma", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "limma",
+        "description": "Data analysis, linear models and differential expression for microarray data.",
+        "source": "http://www.ebi.ac.uk/efo/swo/SWO_0000593"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Source 404s

File: synapseAnnotations/data/analysis.json [hunk 19]
```diff
@@ -1,388 +1,340 @@
...
-      }, 
+        "value": "sleuth",
+        "description": "Differential expression analysis incorporating quantification uncertainty.",
+        "source": "https://www.nature.com/articles/nmeth.4324"
+      },
       {
-        "value": "cuffDiff", 
-        "description": "", 
-        "source": ""
+        "value": "cuffDiff",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Cuffdiff 2

File: synapseAnnotations/data/analysis.json [hunk 20]
```diff
@@ -1,388 +1,340 @@
...
+    "description": "DNA or RNA sequence alignment method",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "BWA", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "BWA",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Class is "Burrows-Wheeler technique". Alternatively BOWTIE.

File: synapseAnnotations/data/analysis.json [hunk 21]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Burrows Wheeler Alignment algorithm.",
+        "source": "http://purl.obolibrary.org/obo/ERO_0100332"
+      },
       {
-        "value": "bowtie", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "bowtie",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Bowtie

File: synapseAnnotations/data/analysis.json [hunk 22]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "A memory efficient short read alignment algorithm.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/19261174"
+      },
       {
-        "value": "bowtie2", 
-        "description": "", 
-        "source": ""
+        "value": "bowtie2",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Bowtie 2

File: synapseAnnotations/data/analysis.json [hunk 23]
```diff
@@ -1,388 +1,340 @@
...
+        "value": "GSNAP",
+        "description": "Method to detect complex variants and splicing in short read sequencing.",
+        "source": "https://academic.oup.com/bioinformatics/article/26/7/873/212606"
+      },
       {
-        "value": "topHat", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "topHat",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: TopHat

File: synapseAnnotations/data/analysis.json [hunk 24]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Method to align short read sequencing for outputs such as splice junction detection.",
+        "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2672628/"
+      },
       {
-        "value": "mosaic", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "mosaic",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: MOSAIC

File: synapseAnnotations/data/analysis.json [hunk 25]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Segmentation alignment method for DNA alignment.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/11238079"
+      },
       {
-        "value": "novoAlign", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "novoAlign",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: NovoAlign

File: synapseAnnotations/data/analysis.json [hunk 26]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "Short read sequencing alignment method.",
+        "source": "http://www.novocraft.com/products/novoalign/"
+      },
       {
-        "value": "olego", 
-        "description": "", 
-        "source": ""
+        "value": "olego",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: OLego

File: synapseAnnotations/data/analysis.json [hunk 27]
```diff
@@ -1,388 +1,340 @@
...
-        "value": "targetValidation", 
-        "description": "", 
-        "source": ""
-      }, 
-      {
-        "value": "CRISPR-CAS9", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "CRISPR-CAS9",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: CRISPR-Cas9

File: synapseAnnotations/data/analysis.json [hunk 28]
```diff
@@ -1,388 +1,340 @@
...
+        "description": "The process in which double-stranded RNAs silence cognate genes. Involves posttranscriptional gene inactivation ('silencing') both of transgenes or dsRNA introduced into a germline, and of the host gene(s) homologous to the transgenes or dsRNA. This silencing is triggered by the introduction of transgenes or double-stranded RNA (dsRNA), and can occur through a specific decrease in the level of mRNA, or by negative regulation of translation, of both host genes and transgenes.",
+        "source": "http://purl.obolibrary.org/obo/GO_0016246"
+      },
       {
-        "value": "shRNA", 
-        "description": "", 
-        "source": ""
+        "value": "shRNA",
+        "description": "Short hairpin RNA (shRNA) can be used to achieve stable knockdown of gene expression. shRNA molecules are cloned into lentiviral or other vectors and expressed by a pol III type promoter.",
+        "source": "http://www.bioassayontology.org/bao#BAO_0000323"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Does this source work for anyone else?

File: synapseAnnotations/data/analysis.json [hunk 29]
```diff
@@ -1,388 +1,340 @@
...
+    "name": "enrichmentMethod",
+    "description": "",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "fishersExactTest", 
-        "description": "", 
-        "source": ""
+        "value": "fishers exact test",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Fisher's exact test

File: synapseAnnotations/data/analysis.json [hunk 30]
```diff
@@ -395,143 +347,138 @@
         "source": "http://edamontology.org/operation_0292"
       },
       {
-        "value": "genotypeImputation", 
-        "description": "The statistical inference of unobserved genotypes", 
-        "source": ""
-      }, 
+        "value": "genotype imputation",
+        "description": "The statistical inference of unobserved genotypes.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/17572673"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Does this method have a name or is it just one instance of "genotype imputation"?

File: synapseAnnotations/data/analysis.json [hunk 31]
```diff
@@ -395,143 +347,138 @@
...
-        "value": "DNAmethylationImputation", 
-        "description": "", 
+        "value": "DNA methylation imputation",
+        "description": "The statistical inference of unobserved DNA methylation marks.",
         "source": "https://dx.doi.org/10.1534/genetics.115.185967"
       },
       {
-        "value": "polygenicRiskScore",
-        "description": "A sample level estimate of the genetic component of disease risk based on genome-wide association studies",
+        "value": "polygenic risk score",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Polygenic Risk Scores

File: synapseAnnotations/data/analysis.json [hunk 32]
```diff
@@ -395,143 +347,138 @@
...
+        "description": "A sample level estimate of the genetic component of disease risk based on genome-wide association studies.",
         "source": "http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003348"
       },
       {
-        "value": "Enrichment Analysis", 
-        "description": "Over-representation analysis", 
+        "value": "enrichment analysis",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Preferred Name "Enrichment analysis"

File: synapseAnnotations/data/analysis.json [hunk 33]
```diff
@@ -395,143 +347,138 @@
...
-      }, 
+      },
       {
-        "value": "differentialExpression", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "statistical network reconstruction",
+        "description": "Learning network relationships between genes based on their coexpression patterns.",
+        "source": "http://bioontology.org/ontologies/BiomedicalResourceOntology.owl#Signaling_Network_Reconstruction"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: 404

File: synapseAnnotations/data/analysis.json [hunk 34]
```diff
@@ -395,143 +347,138 @@
...
+        "source": "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C113415"
+      },
       {
-        "value": "covariateAssociation", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "outlier detection",
+        "description": "An observation in a data set that is numerically distant from the rest of the data.",
+        "source": "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C79083"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: None of these evs.nci.nih.gov links have been working for me.

File: synapseAnnotations/data/analysis.json [hunk 35]
```diff
@@ -395,143 +347,138 @@
...
+      },
       {
-        "value": "hiddenConfounderDetection", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "covariate association",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: covariate specification

File: synapseAnnotations/data/analysis.json [hunk 36]
```diff
@@ -395,143 +347,138 @@
...
+        "value": "principal component analysis",
+        "description": "Principal Component Analysis: Mathematical procedure that transforms a number of possibly correlated variables into a smaller number of uncorrelated variables called principal components.",
+        "source": "http://purl.bioontology.org/ontology/MESH/D025341"
+      },
       {
-        "value": "clustering", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "genome wide association",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Preferred Name "Genome-Wide Association Study"
Synonym "Genome-Wide Association"

File: synapseAnnotations/data/analysis.json [hunk 37]
```diff
@@ -395,143 +347,138 @@
...
+        "description": "An analysis comparing the allele frequencies of all available (or a whole GENOME representative set of) polymorphic markers in unrelated patients with a specific symptom or disease condition, and those of healthy controls to identify markers associated with a specific disease or condition.",
+        "source": "http://purl.bioontology.org/ontology/MESH/D055106"
+      },
       {
-        "value": "supervisedLearning", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "clustering",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Preferred Name "Clustering"

File: synapseAnnotations/data/analysis.json [hunk 38]
```diff
@@ -395,143 +347,138 @@
...
+        "description": "Group together some data entities on the basis of similarities such that entities in the same group (cluster) are more similar to each other than to those in other groups (clusters).",
+        "source": "http://edamontology.org/operation_3432"
+      },
       {
-        "value": "batchEffectCorrection", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "supervised learning",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Preferred Name "Supervised Machine Learning"

File: synapseAnnotations/data/analysis.json [hunk 39]
```diff
@@ -395,143 +347,138 @@
...
+        "value": "batch effect correction",
+        "description": "Adjustment of variable of interest for batch effects.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/16632515"
+      },
       {
-        "value": "variancePartition", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "anova",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: ANOVA

File: synapseAnnotations/data/analysis.json [hunk 40]
```diff
@@ -395,143 +347,138 @@
...
+        "description": "ANOVA or analysis of variance is a data transformation in which a statistical test of whether the means of several groups are all equal.",
+        "source": "http://purl.obolibrary.org/obo/OBI_0200201"
+      },
       {
-        "value": "moduleIdentification", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "variance partition",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: variancePartition

File: synapseAnnotations/data/analysis.json [hunk 41]
```diff
@@ -395,143 +347,138 @@
...
+        "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5123296/"
+      },
       {
-        "value": "visualization", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "visualization",
+        "description": "Resource that provides access to tools for graphical, interactive, multimodal and multidimensional data and result visualization.",
+        "source": "http://bioontology.org/ontologies/BiomedicalResourceOntology.owl#Visualization"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: 404

File: synapseAnnotations/data/analysis.json [hunk 42]
```diff
@@ -395,143 +347,138 @@
...
       },
       {
-        "value": "Copy number estimation",
+        "value": "copy number estimation",
         "description": "Estimate the number of copies of loci of particular gene(s) in DNA sequences typically from gene-expression profiling technology based on microarray hybridisation-based experiments.",
         "source": "http://edamontology.org/operation_3233"
       },
       {
-        "value": "Gene expression profile comparison",
+        "value": "gene expression profile comparison",
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Preferred Name: "Gene expression comparison"

File: synapseAnnotations/data/analysis.json (hunk 18 above)
Comment (sgosline) [reply synapseAnnotations/data/analysis.json]: This is an issue with SWO, @kdaily created an issue on their repo. For now can you put the full query link?

File: synapseAnnotations/data/analysis.json [hunk 43]
```diff
@@ -568,106 +515,88 @@
...
+        "description": "A classification model that assumes independence of predictors and uses Bayes theorem predict outcomes.",
+        "source": "http://engr.case.edu/ray_soumya/mlrg/optimality_of_nb.pdf"
+      },
       {
-        "value": "GaussianProcessRegression", 
-        "description": "", 
-        "source": ""
+        "value": "Gaussian process regression",
+        "description": "A non parametric kernel based supervised learning regression method.",
+        "source": "http://www.inference.org.uk/itprnn/book.pdf"
```
Comment (philerooski) [inline synapseAnnotations/data/analysis.json]: Ah, the classic "just cite the whole textbook" technique

Comment (sgosline) [reply synapseAnnotations/data/analysis.json]: So we can just cite Judea Pearl for anything Bayesian, while we're at it, @blogsdon ?

Comment (blogsdon) [reply synapseAnnotations/data/analysis.json]: Wait, in the contributing docs does it explicitly say you can't cite a text book?

Comment (kdaily) [reply synapseAnnotations/data/analysis.json]: It does not! I would, however, make this a proper citation. I don't have a specific formatting preferences, but something that at least gets the  author, year, edition, page number in there. I would also link to http://www.inference.org.uk/mackay/itila/ instead.

File: synapseAnnotations/data/analysis.json (hunk 32 above)
Comment (sgosline) [reply synapseAnnotations/data/analysis.json]: But that is not what you put in the file....

**Pull request merged**